CONTRIBUTING.md

```diff
@@ -57,6 +57,18 @@ pnpm run dev
 pnpm run build
 ```
 
+**How can I test my changes while contributing to the repository?**
+
+During the development process, you may want to test your changes to ensure they are working as expected. Here are a few ways to do it:
+
+1. Run any of the examples in the examples folder. They are linked to use the local Astro, so you can see the effects of your changes.
```

Comment:
Nit, I'd like to clarify here we're talking about the source code itself to avoid confusion.
```suggestion
1. Run any of the examples in the `/examples` folder. They are linked to use the local Astro source code, so you can see the effects of your changes.
```